.eslintrc.json

```diff
@@ -6,10 +6,6 @@
     "extends": [
         "airbnb-base"
     ],
-    "globals": {
```

Comment:
So this weirdness is totally a bug, but as you noted, it's just a bug with passing the config via the cli.  If eslint loads the config itself from the known config location, the bug isn't hit.  Coincidentally, the rollup-pluglin-eslint extra options can't be passed if the config is specified (it can only be either a config name string or an object), even though they have nothing to do with one another.  So instead, I ended up falling back to the default config location (which actually works fine) and passing in the 'throwOnError:true' config parameter to ensure the rollup build will actually fail when linting fails.